go/src/koding/kites/kloud/provider/azure/stack.go

```diff
@@ -99,6 +101,14 @@ func (s *Stack) VerifyCredential(c *stack.Credential) error {
 	return nil
 }
 
+func trim(s string, n int) string {
```

Comment:
Creating a *bytes.Buffer just to get a slice of a string? I think it's not needed.